pipeline/run_eval.py

```diff
@@ -123,14 +123,20 @@ def get_sampling_cmd(benchmark, random_seed, extra_eval_args="", extra_arguments
         default="",
         help="Any extra arguments to pass to nemo_skills/evaluation/evaluate_results.py",
     )
+
     args, unknown = parser.parse_known_args()
 
     extra_arguments = f'{" ".join(unknown)}'
 
     args.model_path = Path(args.model_path).absolute()
     args.output_dir = Path(args.output_dir).absolute()
 
-    server_start_cmd, num_tasks = get_server_command(args.server_type, args.num_gpus)
+    server_start_cmd, num_tasks = get_server_command(args.server_type, args.num_gpus, args.model_path.name)
+
+    if args.server_type == "vllm":
+        server_wait_string = "Uvicorn running"
```

Comment:
Can you return this from the ger_server_command, so that we keep it in one place?